actions.py

```diff
@@ -5,8 +5,7 @@
 
 
 class YearlyExtremesAction(argparse.Action):
```

Comment:
Why do we need argparse.Action now? We're performing actions through flask app

Reply:
This is the code from the first task where command line was being used, so it's not being used for the django app. The first task's code is also in this repo, its refactored.